app/javascript/src/components/TaxStatus.vue

```diff
@@ -17,43 +17,50 @@
     <div v-if="showTaxStatus">
       <div class="flex justify-between">
         <span class="w-1/3"></span>
-        <span class="w-1/3">Power points</span>
-        <span v-if="game.baseGame === 'imperial'" class="w-1/3">Tax chart position</span>
+        <span class="w-1/5">Flags</span>
```

Comment:
Should this be both Flags and Factories? Is it useful to know how many flags there are _without_ also rolling in the factory (x2) count?

Reply:
I'm not too sure how to adjust the adjust the chart width, right now there's limited space and each column is pretty packed to another.

For now I've added only the flags, since usually it's easier to count factories (1-5), lmk how to proceed

Reply:
Ah, this is the PR where it isn't as crowded yet, check the screenshots in the next PR I think